File: app/src/main/java/com/electrosim/simulation/InputPort.java
/**
 * 
 */
package com.electrosim.simulation;

import com.electrosim.ElementInterface;

/**
 * @author HANNAN
 *
 */
public class InputPort extends SimPort {

	/**
	 * @param x
	 * @param y
	 * @param radius
	 * @param container
	 */
	public InputPort(float x, float y, float radius, ElementInterface container) {
		super(x, y, radius, container);
	}

	public InputPort(float x, float y,ElementInterface container) {
		super(x, y, container);
	}

}

File: app/src/main/java/com/electrosim/simulation/InputPort.kt
/**
 * 
 */
package com.electrosim.simulation

import com.electrosim.ElementInterface

/**
 * @author HANNAN
 *
 */
class InputPort : SimPort {
    /**
     * @param x
     * @param y
     * @param radius
     * @param container
     */
    constructor(x: Float, y: Float, radius: Float, container: ElementInterface) : super(x, y, radius, container)

    constructor(x: Float, y: Float, container: ElementInterface) : super(x, y, container)
} 